config/locales/en.notifier.yml

```diff
@@ -5,9 +5,18 @@ en:
         subject: "!app_name! Payment Method Updated"
       new_user:
         intro_html: "Congratulations, %{first_name}. A !app_name! account has been created for you. You may log in at <a href=\"%{login_url}\">%{login_url}</a>."
```

Comment:
Protip: with the markdown parser, you can use it's syntax for links: `You may log in at [%{login_url}][%{login_url}].` (there's a chance I have the parens/square brackets reversed:).